upgrade/migrations.go

```diff
@@ -24,7 +24,7 @@ const (
 func AutoAliasingMigration(resourcesFilePath, providerName string) (bool, error) {
 	// Create the AST by parsing src
 	fset := token.NewFileSet()
-	file, err := parser.ParseFile(fset, resourcesFilePath, nil, parser.ParseComments)
+	file, err := parser.ParseFile(fset, resourcesFilePath, nil, parser.ParseComments|parser.SkipObjectResolution)
```

Comment:
Syntactic object resolution is deprecated now including `ast.Object`, see: https://pkg.go.dev/go/ast#Object.

We don't need object resolution because we're doing ast manipulation and not analysis or resolving symbols across scopes.